src/org/zaproxy/zap/utils/HirshbergMatcher.java

```diff
@@ -26,7 +26,7 @@
  * or the similarity ratio between two strings.
  * This implementation is using Hirschber's algorithm B and algorithm C.
  *
- * @see <a href="http://code.google.com/p/algorithm800/source/browse/trunk/src/Hirschberg.java">Hirschberg&apos;s algorithm
+ * @see <a href="http://code.google.com/p/algorithm800/source/browse/trunk/src/Hirschberg.java">Hirschberg's algorithm
```

Comment:
Well no big deal then, links will die over time people will just have to accept that.